src/annotator/adder.js

```diff
@@ -83,24 +83,23 @@ export class Adder {
    *
    * The adder is initially hidden.
    *
-   * @param {HTMLElement} container - The DOM element into which the adder will be created
+   * @param {HTMLElement} element - The DOM element into which the adder will be created
    * @param {AdderOptions} options - Options object specifying `onAnnotate` and `onHighlight`
    *        event handlers.
    */
-  constructor(container, options) {
-    this._container = container;
-    this._shadowRoot = createShadowRoot(container);
+  constructor(element, options) {
+    this._outerContainer = document.createElement('hypothesis-adder');
```

Comment:
I see that the naming of the `element` argument here and `this._outerContainer` is consistent with the `Notebook` class. I think it would be good to change the sidebar in future to also be consistent. One way to do this might be to extract a pattern (see below).